pkg/cfn/builder/vpc.go

```diff
@@ -71,9 +76,24 @@ func (c *ClusterResourceSet) addSubnets(refRT *gfn.Value, topology api.SubnetTop
 	}
 }
 
+func (c *ClusterResourceSet) addNATGateways() {
+
+	switch c.spec.VPC.NATGateway {
+
+	case "highly-available":
```

Comment:
Kubernetes tradition would be to use string constants of following form: `HighlyAvailable`, `Single`, `Disable`.